src/BookNavigator/styles/button-base.js

```diff
@@ -6,7 +6,7 @@ export default css`
     border: none;
     outline: none;
     cursor: pointer;
-    color: #fff;
+    color: var(--primaryTextColor, #fff);
```

Comment:
I guess we don't want the defaults from the children components; they expect to be defined.